File: Python/Conditional Statement/grade.py
per = float(input("Enter your percentage:"))

if per<=100 and per>=91:
    print("Grade A")
elif per<91 and per>=81:
    print("Grade B")
elif per<81 and per>=71:
    print("Grade C")
elif per<71 and per>=61:
    print("Grade D")
elif per<61 and per>=40:
    print("Grade E")
elif per<40:
    print("FAILED")
else:
    print("INVALID PERCENTAGE")

File: C/Looping/grade.c
# include <stdio.h>

int main (){
	
	int n,m,roll,i,j,total=0,marks;
	float per;
	
	printf("How many students are there in class?:");
	scanf("%d",&n);
	
	printf("How many subjects do they learn?:");
	scanf("%d",&m);
	
	
	for (i=1; i<=n; i++){
		printf("Please enter students roll number:");
		scanf("%d",&roll);
		total=0;
		for (j=1; j<=m; j++){
			printf("Please enter your marks:");
			scanf("%d",&marks);
			
			total = total+marks;
		}
		
		per = (float)total/m;
		
		printf("Roll No.:%d : Percentage is:%f\n",roll,per);
		
		if(per>=85 && per<=100){
			printf("Grade:A\n");
		} else if (per>=75 && per<85){
			printf("Grade:B\n");
		} else if (per<75) {
			printf("Grade:C\n");
		}
	}
	
	return 0;
}
